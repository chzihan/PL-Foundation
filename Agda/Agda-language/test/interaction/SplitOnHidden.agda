data Nat : Set where
  zero : Nat
  suc : Nat → Nat

test : ∀{N M : Nat} → Nat → Nat → Nat
test N M = {!.N N .M!}
-- Andreas, 2016-07-10, issue 2088
-- Changed behavior:
-- The hidden variables .N and .M are made visible
-- only the visible N is split.
