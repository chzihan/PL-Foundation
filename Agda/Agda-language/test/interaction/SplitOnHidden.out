(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : Nat " nil)
((last . 1) . (agda2-goals-action '(0)))
((last . 2) . (agda2-make-case-action '("test {N} {M₁} zero M = ?" "test {N} {M₁} (suc N₁) M = ?")))
((last . 1) . (agda2-goals-action '(0)))
(agda2-info-action "*Error*" "SplitOnHidden.agda:6,12-23 Unbound variable .x when checking that the expression ? has type Nat" nil)
((last . 3) . (agda2-maybe-goto '("SplitOnHidden.agda" . 103)))
(agda2-highlight-load-and-delete-action)
(agda2-status-action "")
